src/nvim/testdir/test_regexp_utf8.vim

```diff
@@ -590,4 +590,12 @@ func Test_match_char_class_upper()
   bwipe!
 endfunc
 
+func Test_match_invalid_byte()
+  call writefile(0z630a.765d30aa0a.2e0a.790a.4030, 'Xinvalid')
+  new
+  source Xinvalid
+  bwipe!
+  call delete('Xinvalid')
+endfunc
+
```

Comment:
I'll get rid of this. This was a added by mistake (not sure how).

Reply:
maybe a messed up rebase? 🤷

Reply:
I think so but I'm not sure.

I dropped the commit that was mistakely added.